test/global.test.ts

```diff
@@ -5,12 +5,16 @@
 
 import { IHookCallbackContext } from 'mocha';
 import * as vscode from 'vscode';
+import { TestUserInput } from 'vscode-azureextensiondev';
+import { ext } from '../extension.bundle';
 
 // tslint:disable-next-line:strict-boolean-expressions export-name
 export let longRunningTestsEnabled: boolean = !/^(false|0)?$/i.test(process.env.ENABLE_LONG_RUNNING_TESTS || '');
+export const testUserInput: TestUserInput = new TestUserInput(vscode);
 
 // Runs before all tests
 suiteSetup(async function (this: IHookCallbackContext): Promise<void> {
     this.timeout(120 * 1000);
     await vscode.commands.executeCommand('cosmosDB.refresh'); // activate the extension before tests begin
+    ext.ui = testUserInput;
```

Comment:
Add `ext.outputChannel = new TestOutputChannel()` here.  We want to see the output being displayed when the tests run.